### ISSUE TITLE
Error while rnv is listing Android target devices

### ISSUE THREAD

**Issue by hosek** [pavjacko/renative#129]
Cant deploy to device/emulator due this:
```
ReNative run - _listAndroidTargets:android:false:false:false - Starting!
execCLI:null:echo "avd name" | nc localhost 5554
execCLI:null:echo "avd name" | nc localhost 5556

ReNative run - ERRROR! Cannot read property 'slice' of undefined
TypeError: Cannot read property 'slice' of undefined
```
platformTools/android.js:162

also please note typo "ERRROR"

**Comment by mihaiblaga89** [comment]
What version of rnv do you have?

**Comment by hosek** [comment]
Oh sorry, I had 0.23.17.
Retesting it now with latest version 0.23.19 gives me _ReNative run - ERRROR! No simulator -t target name specified!_ also not all emulators (3 of 6) are listed in CLI

**Comment by hosek** [comment]
Fixed in 0.23.20, closing
